### PR TITLE
Add link to Optional STH Witness feature

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**

### CODE REVIEW

File: binary_transparency/firmware/docs/design/README.md [hunk 1]
```diff
@@ -234,7 +234,8 @@ _discoverability_ that Firmware Transparency leverages.
 
 **STH Witness**:
 
-_Not yet implemented here._
+May use the [`ft_witness`](/binary_transparency/firmware/cmd/ft_witness) to
+support verfication of log operator, for consistent log operation:
```
Comment (AlCutter) [inline binary_transparency/firmware/docs/design/README.md:238]: how about `... to enable detection of log split views`?

wdyt?

Comment (yogeshbdeshpande) [reply binary_transparency/firmware/docs/design/README.md:238]: Yes, I was also planning to add something similar, however we have not explained the terms "split views and split view attacks till now, so was unsure. They have been explained in the STH Witness Page..

Comment (yogeshbdeshpande) [reply binary_transparency/firmware/docs/design/README.md:238]: Perhaps I can add:
to support additional verification of log operator by enabling detection of log split views.

Comment (yogeshbdeshpande) [reply binary_transparency/firmware/docs/design/README.md]: Please check this revision, with some corrections, on lines similar to what you had suggested!

**Review by AlCutter — APPROVED** [review]

**Pull request merged**